<a href="https://colab.research.google.com/github/ucfilho/Biochemical_Engineering/blob/master/Biochemical_Engineering/Courses/EngBioq_2024_01/Aula_04_agitacao_mistura_jan_15_2024_B.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

## Exemplo 2
Calcule o gasto energético associado a um reator de 600 m$^3$ considere N$_P$=1.68, $\rho$=1200 kg/m$^3$, $\mu$=0.001 kg/m.s e o custo de 1kWh vale $0.98. Considere as duas situações: Nrps =0.5 e Nrps =0.75

In [1]:
pi = 3.14159
Vol =600
Np = 1.68
ro =1200
mi = 0.001
Nrps = [0.5,0.75]
tempo = 30*24*60*60
kWh = 0.98 # $/kWh

In [2]:
d = (4*Vol/pi)**(1/3)
di = d/3
print('d: {:.2f} di: {:.2f}'.format(d, di))

d: 9.14 di: 3.05


In [3]:
for Ni in Nrps:
  Re = ro*di**2*Ni/mi
  Pot = Np*ro*di**5*Ni**3
  Gasto = Pot*tempo/3.6e6*kWh
  print('Nrps:{:.2f} Re:{:.2e} Pot:{:.2f} Gasto:{:.2f}'.format(Ni,Re,Pot,Gasto))


Nrps:0.50 Re:5.57e+06 Pot:66205.97 Gasto:46714.93
Nrps:0.75 Re:8.36e+06 Pot:223445.15 Gasto:157662.90


Nrps:0.50 Re:5.57e+06 Pot:66205.97 Gasto:46714.93 <br>
Nrps:0.75 Re:8.36e+06 Pot:223445.15 Gasto:157662.90

## Exemplo 3
Calcule o tempo de mistura para duas situações do exemplo 2. Considere N$_Q$=1.2 e a mesmas duas situações: Nrps =0.5 e Nrps =0.75

In [4]:
Nq= 1.2
for Ni in Nrps:
  Q = Nq*Ni*di**3
  tm = 3*Vol/Q # valor em segundos
  tm = tm/60 # passando para minutos
  print('Nrps:{:.2f} Q:{:.2f} tm:{:.2f}'.format(Ni,Q,tm))

Nrps:0.50 Q:16.98 tm:1.77
Nrps:0.75 Q:25.46 tm:1.18


## Exemplo 4
Calcule o gasto energético associado a um reator de 1200 m$^3$ considere N$_P$=1.68, $\rho$=1200 kg/m$^3$, $\mu$=0.001 kg/m.s e o custo de 1kWh vale $0.98. Considere as duas situações: Nrps =0.5 e Nrps =0.75

In [5]:
pi = 3.14159
Vol =1200
Np = 1.68
ro =1200
mi = 0.001
Nrps = [0.5,0.75]
tempo = 30*24*60*60
kWh = 0.98 # $/kWh
d = (4*Vol/pi)**(1/3)
di = d/3
print('d: {:.2f} di: {:.2f}'.format(d, di))
for Ni in Nrps:
  Re = ro*di**2*Ni/mi
  Pot = Np*ro*di**5*Ni**3
  Gasto = Pot*tempo/3.6e6*kWh
  print('Nrps:{:.2f} Re:{:.2e} Pot:{:.2f} Gasto:{:.2f}'.format(Ni,Re,Pot,Gasto))

d: 11.52 di: 3.84
Nrps:0.50 Re:8.84e+06 Pot:210190.86 Gasto:148310.67
Nrps:0.75 Re:1.33e+07 Pot:709394.14 Gasto:500548.51
